piker/data/feed.py

```diff
@@ -405,14 +382,26 @@ async def start_backfill(
                 # request loop until the condition is resolved?
                 return
 
-            if next_start_dt in starts:
+            if (
+                next_start_dt in starts
+                and starts[next_start_dt] <= 6
```

Comment:
magic number

Reply:
yeah i mean, i just tend to use 6s and 1s and there's no reason. It's literally for random choices that the universe favors.

I mean why else would we try 6 times.. i dunno